<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDateSetDialog</class>
 <widget class="QDialog" name="TimeDateSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>日期设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButtonSetTime">
          <property name="text">
           <string>设置时间</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButtonSetDateTime">
          <property name="text">
           <string>设置日期时间</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="displayFormat">
           <string>HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEditDateTime"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEditTime"/>
        </item>
        <item row="5" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="displayFormat">
           <string>yyyy/MM/dd HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButtonSetDate">
          <property name="text">
           <string>设置日期</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>时间日期</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="displayFormat">
           <string>yyyy/MM/dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditDate"/>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QPushButton" name="pushButtonGetCurTime">
          <property name="text">
           <string>获取当前日期时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>定时器</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonTimeBegin">
            <property name="text">
             <string>开始计时</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTimeStop">
            <property name="text">
             <string>停止计时</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>用时</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditElapse"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>定时周期</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxElapse">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSetPeriod">
            <property name="text">
             <string>设置周期</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,3,1,3">
          <item>
           <widget class="QLCDNumber" name="lcdNumberHour">
            <property name="digitCount">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>32</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberMinute">
            <property name="digitCount">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>32</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberSecond">
            <property name="digitCount">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>日历</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>选择的日期</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditChoiceCalendar"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="gridVisible">
           <bool>false</bool>
          </property>
          <property name="navigationBarVisible">
           <bool>true</bool>
          </property>
          <property name="dateEditEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCurDateTime">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>TimeDateSetDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
